### PR TITLE
feat : added results page component

### PR DESCRIPTION
**Pull request by Priya-Patel31**
It consists of :
* Header
* Main-container
 • Sidebar
 • results page container

### CODE REVIEW

File: results_page/results_page.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                <li class="list-item">Toubkal</li>
+                <li class="list-item">Simien</li>
+              </ol>
+            </div>
+            <p class="my-1 correct-answer font-medium">Correct Answer : Mount Kenya </p>
+            <p class="my-1 wrong-answer font-medium">Your Answer : Simien</p>
+          </div>
+          <div class="space-2rem"></div>
+          <button class="retake-quiz-btn font-bold">Retake Quiz </button>
+          <div class="space-2rem"></div>
```
Comment (Ankur9669) [inline results_page/results_page.html:105]: Suggestion: Can add some hover effect on the button

**Review by Ankur9669 — APPROVED** [review]
The UI looks good. Just a small suggestion you can add links in the navbar. And the navbar is not responsove below 400px.

But it looks good to me

File: results_page/results_page.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (t007rushi) [inline results_page/results_page.html]: just title change required

**Review by t007rushi — APPROVED** [review]
Logout hover text is not visible as it gets the same color as bg need to look for all pages

File: results_page/results_page.html (hunk 2 above)
Comment (Priya-Patel31) [reply results_page/results_page.html]: yes i have changed it !!

File: results_page/results_page.html (hunk 1 above)
Comment (Priya-Patel31) [reply results_page/results_page.html:105]: I'll add it !!

**Pull request merged**